account_payment_order/tests/test_payment_order_inbound.py

```diff
@@ -136,3 +137,25 @@ def test_creation(self):
         payment_order.cancel2draft()
         payment_order.unlink()
         self.assertEqual(len(self.payment_order_obj.search(self.domain)), 0)
+
+    def test_payment_partner_bank_value(self):
+        payment_order = self.inbound_order
+        self.assertEqual(len(payment_order.ids), 1)
+
```

Comment:
Better to not add empty lines inside the method for new code. It adds more scroll with no real value.

Reply:
Done